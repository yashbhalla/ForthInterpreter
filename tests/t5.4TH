5.0 DUP * .
