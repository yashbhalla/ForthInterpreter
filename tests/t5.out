Stack not empty: [25.0]
